general/1_terminal/Maksym-Omelianenko/task3.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
```

Comment:
Good job on reading about `set -e`, this is widely used in production scrips.
I'd recommend using `set -ex`, so that the commands that are being executed are printed. More [here](https://www.peterbe.com/plog/set-ex).